### ISSUE TITLE
Typescript types are not in the package

### ISSUE THREAD

**Issue by BryanCrotaz** [pzuraq/tracked-maps-and-sets#16]
PR #15

**Comment by pzuraq** [comment]
sorry for the delay here, I'm getting this whenever I try to build and publish locally:

```
Build Error (broccoli-persistent-filter:Babel > [Babel: @ember/test-helpers]) in @ember/test-helpers/-tuple.js

Cannot find module '/Users/cgarrett/Code/pzuraq/tracked-maps-and-sets/node_modules/@babel/compat-data/data/corejs3-shipped-proposals'


Stack Trace and Error Report: /var/folders/3z/hqmhkq4x307_mnsbrt2yql6800101v/T/error.dump.14e6092801d9e0ecb59564a5b5bbeecb.log
```

I haven't had a chance to dig in and try to figure out what's wrong

**Comment by pzuraq** [comment]
Was able to get it fixed and published in v2.2.1!
